svelte/src/lib/UplotSvelte.svelte

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+    import 'uplot/dist/uPlot.min.css';
+    import type uPlot from 'uplot';
+    import {onDestroy, onMount} from "svelte";
```

Comment:
Please, run prettier on the code-base, it's doesn't seem to be formatted to me. You can use the .prettierrc config in the root directory.